parm/post/oceanice_products_gfs.yaml

```
@@ -1,16 +1,19 @@
 ocnicepost:
   executable: "ocnicepost.x"
   namelist:
-    write_grib2: True
-    write_netcdf: False
+    write_grib2: {{ write_grib2 }}
+    write_netcdf: {{ write_netcdf }}
```

This should stay `False`.  When this is `True`, the program will run and create the interpolated netcdf files, but they remain in `DATA`, which gets removed when the job ends.
```suggestion
    write_netcdf: False
```

---

these variables are now defined in https://github.com/NeilBarton-NOAA/global-workflow/blob/ocnice_products/dev/parm/config/gfs/config.oceanice_products